code/style.css

```diff
@@ -1,14 +1,15 @@
 /* Global css variables used for colors */
 :root {
-  --primary: #a259ff;
-  --secondary: #b0a6ff;
+  --primary: #7a3f78;
+  --secondary: #3a1439;
 }
 
 body {
   display: flex;
   margin: 0;
   padding: 0;
   font-family: 'Montserrat', sans-serif;
+  background: black;
```

Comment:
This is a big too strong a contrast for me personally against the also dark background of the cards. I would consider changing either this or the card background color to something lighter so they stand out better against the dark.